.sail/Dockerfile

```diff
@@ -8,12 +8,12 @@ RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | b
 
 LABEL project_root "~/go/src/go.coder.com"
 
-# Modules break much of Go's tooling.
-ENV GO111MODULE=off
+ENV GO111MODULE=on
```

Comment:
I think this should be submitted in a separate PR for discussion as this is a controversial change. @ammario might have something to say about it, since he was the one who initially added this according to git blame: https://github.com/cdr/sail/pull/49/files#diff-e963b843e440ee24f53402574e0e3dd6.